src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -203,29 +285,23 @@ export const ReaderRevenueBanner = ({ meta, module }: Props) => {
 				setBanner(() => bannerModule[module.name]); // useState requires functions to be wrapped
 				sendOphanComponentEvent('INSERT', meta);
 			})
-			.catch((error) =>
-				// eslint-disable-next-line no-console
-				console.log(`banner - error is: ${error}`),
-			);
-		// eslint-disable-next-line react-hooks/exhaustive-deps
+			.catch((error) => console.log(`banner - error is: ${error}`));
 	}, []);
 
-	// Should only run once
-	useEffect(() => {
-		if (hasBeenSeen && meta) {
+	useOnce(() => {
+		if (hasBeenSeen) {
```

Comment:
You don't need this check here. The way `useOnce` works is it will not run until all dependencies are truthy. So if code execution reaches this point `hasBeenSeen` is guarenteed to be true.

Reply:
Thanks!